packages/core/src/test/amazonqDoc/utils.ts

```diff
@@ -192,16 +196,33 @@ export function createExpectedEvent(params: EventParams) {
     }
 }
 
+export function createExpectedMetricData(operationName: string, result: string) {
+    return {
+        metricName: 'Operation',
+        metricValue: 1,
+        timestamp: new Date(),
+        product: 'DocGeneration',
+        dimensions: [
+            {
+                name: 'operationName',
+                value: operationName,
+            },
+            {
+                name: 'result',
+                value: result,
+            },
+        ],
+    }
+}
+
 export async function assertTelemetry(params: {
```

Comment:
Are these changes to `assertTelemetry` going to be needed every time `/doc` adds an event and type? Likely out of scope here, but can this be refactored to avoid this?